core/templates/pages/exploration-editor-page/exploration-editor-page.component.spec.ts

```diff
@@ -596,7 +652,7 @@ describe('Exploration editor page component', function() {
 
     it('should react to initExplorationPage broadcasts', fakeAsync(() => {
       $scope.$apply();
-
+      spyOn(cns, 'startCheckingConnection');
```

Comment:
WHy do we need this spyOn here?

Reply:
As startCheckingConnection is a periodic timer so we need this spyon to catch that timer in fakyAsync functions.